apps/hosts/utils.py

```diff
@@ -1,6 +1,13 @@
 from .models import ChallengeHost
+from challenges.models import Challenge
```

Comment:
This is just to maintain a better history. Can we please move this in a separate PR?